Sources/Kafka/KafkaConsumer.swift

```diff
@@ -12,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+import Dispatch
```

Comment:
Don't think we need this import here

Reply:
Whoops. Left over from a refactor. Is there any linter that can help point out unused imports?